### ISSUE TITLE
IntegrationTest.php does not comply with psr-4 autoloading standard.

### ISSUE THREAD

**Issue by featuriz** [doctrine/DoctrineFixturesBundle#310]
Now after composer update -o on a simple symfony5 latest application throws error as. Here I post complete terminal output.

```
PS E:\WORK\FZCMS> composer update -o
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "^5.0"
Nothing to install or update
Generating optimized autoload files
Deprecation Notice: Class Doctrine\Bundle\FixturesBundle\Tests\IntegrationTest\IntegrationTest located in E:/WORK/FZCMS/vendor/doctrine/doctrine-fixtures-bundle\Tests\IntegrationTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar://C:/Users/Sudhakar/AppData/Local/ComposerSetup/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Doctrine\Bundle\FixturesBundle\Tests\IntegrationTest\IntegrationTestKernel located in E:/WORK/FZCMS/vendor/doctrine/doctrine-fixtures-bundle\Tests\IntegrationTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar://C:/Users/Sudhakar/AppData/Local/ComposerSetup/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
62 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Executing script cache:clear [OK]
Executing script assets:install public_html [OK]

PS E:\WORK\FZCMS>
```

**Comment by featuriz** [comment]
![symfony-eror (2)](https://user-images.githubusercontent.com/10089154/79043022-1d40c780-7c1a-11ea-9092-2d02cc6d17fa.jpg)

**Comment by greg0ire** [comment]
I just released the fix made in #306 as https://github.com/doctrine/DoctrineFixturesBundle/releases/tag/3.3.1 . Please use that.

**Comment by featuriz** [comment]
Okay, Its working now. Thanks for your fix. Also I love that I have done some contribution to this project.
